tests/test_logger.py

```diff
@@ -25,6 +25,7 @@ def _compare_csv_lines(csv_path: str, expect: dict):
 
 def test_no_accum(tmpdir):
     logger.configure(tmpdir, ["csv"])
+    sb_logger.record("A", -1)
```

Comment:
Adding this line checks that the recorded "A": -1 is overwritten by  "A": 1 in the next line.

Previously, the observed value would be the mean of these two values (0) instead.

Reply:
This might be worth adding as a Python comment?

Reply:
added